app/models/vote.rb

```diff
@@ -29,8 +29,12 @@ def self.noes
     where("votes.vote = 'no'")
   end
 
-  def rebellion?
-    !free? && vote != whip_guess
+  def rebellion?(whip=nil)
```

Comment:
Style-only: [spaces around the `=` are preferred](https://github.com/bbatsov/ruby-style-guide#spaces-around-equals).